waterlog/src/Components/BtnResolve.js

```diff
@@ -1,29 +1,33 @@
- import React, { Component } from "react";
-import PropTypes from "prop-types";
-import { connect } from "react-redux";
-import { fetchSegmentsLeaksResolve } from "../actions/SegmentResolveActions";
- 
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchSegmentsLeaksResolve } from '../actions/SegmentResolveActions';
+import { Globals } from './../Globals';
 
 class BtnResolve extends Component {
+
   constructor(props) {
     super(props);
     this.state = {
       text: "Resolve Issue"
     };
   }
+
   componentDidMount() {
     this.props.fetchSegmentsLeaksResolve(this.props.id);
   }
+
   render() {
     const { error, loading, leaksResolves } = this.props;
-
+    console.log({ leaksResolves });
```

Comment:
do you want this console.log to be in Dev for testing purpose? or you forgot it by mistake?

Reply:
I forgot it by mistake. I removed it.